server/tests/test_sse.py

```diff
@@ -86,6 +85,35 @@ async def test_missed_events(minimalorg: MinimalorgRpcClients, backend: Backend)
         )
 
 
+async def test_empty_last_event_id(minimalorg: MinimalorgRpcClients, backend: Backend) -> None:
+    """
+    When the client provide `Last-Event-ID` with an empty value,
+    the server should ignore it and keep processing as if no `Last-Event-ID` was provided.
```

Comment:
BTW: header is sometimes referred as `Last-Event-ID` and sometimes as `Last-Event-Id` in our code base. This is not much of an issue because header field names are case-insensitive, but it would be nice to use the same case everywhere.

Reply:
The spec write `Last-Event-ID` with `ID` in full caps: https://html.spec.whatwg.org/multipage/server-sent-events.html#the-last-event-id-header